src/main/java/seedu/address/model/applicant/Qualification.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.applicant;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's name in the address book.
```

Comment:
Should be Applicant's Qualification here?

Reply:
Fixed the error, please review again.